QA Sprint 156 done

EZheln opened the pull request:
I
Updated tests for Train model wizard
- Jira: ML-5328, ML-5234, ML-5333

II
Updated tests according to requirements
- changed uniqueness validation and for Register artifact, Register dataset
- update messages for Register artifact, Register dataset
- updated mock server functions related to edit and delete tags according to new implementation
- removed the label value validation rules Batch Run wizard
- changed message Tag to Version Tag feature store, feature vector
- added check label validation rules on Create new project pop-up
- Jira: ML-4927, ML-4916, ML-5440, ML-4835

viktor-ilchuk committed:
Fix [Feature Store] Target store checkboxes don't work properly (#1917)

Co-authored-by: Viktor Ilchuk <viktor.ilchuk@amcnetworks.com>

ilan7empest committed:
Update DRC to `1.4.0` (#1930)

mavdryk committed:
Fix [Batch Run] Number input for int and float parameters is malfunctioning (#1918)

mariana-furyk committed:
Fix [Combobox] unable to pass model as data input using store path (#1921)

mariana-furyk committed:
Fix [Features Analysis] Large float values are not presented (#1923)

illia-prokopchuk committed:
Impl [Model endpoint] UI should display drift threshold and drift actual value (#1925)

mavdryk committed:
Fix [Batch run] Hide 'Build image' option, add message for 'Image name' (#1927)

1. Hide 'Build a new image' option for 'Batch inference' and 'Batch run'
2. Add an explanation message for 'Image name' input

mavdryk committed:
Fix [Batch run] Changed parameter types aren't saved for rerun (#1931)

illia-prokopchuk committed:
Fix [Jobs] UI is stuck on attempt to list runs for job with many runs (#1934)

ilan7empest committed:
Fix [Scheduled workflow] Workflow (and not job) should have different icon and minimal editing (#1935)

illia-prokopchuk committed:
Fix [Jobs] Batch Model Endpoint => overview - model, function and Featureset links are broken (#1937)

illia-prokopchuk committed:
Fix [Model Endpoints] Invalid Function URI link in model endpoints screen (#1939)

illia-prokopchuk committed:
Fix [Feature set] the target store path is incorrect (#1942)

mavdryk committed:
Fix [Batch run] Mandatory parameters should be selected (#1946)

mariana-furyk committed:
Fix [Batch run] Remove the '$' from Batch Inference version presentation (#1949)

mavdryk committed:
Fix [Batch run] Batch run labels are incorrect (#1952)

ilan7empest committed:
Bump DRC version to `1.4.1` (#1957)

mavdryk committed:
Impl [Wizard steps] Next to step should be clickable (#1943)

ilan7empest committed:
Fix [Batch Run] UI minor fix (#1953)

ilan7empest committed:
Impl [Batch run] Add "monitoring" to categories mapping (#1955)

mariana-furyk committed:
Fix [Batch run] inputs issues (#1961)

mariana-furyk committed:
Fix [Datasets and Models] Filter by tag isn't performed when version tag was edited (#1963)

ilan7empest committed:
Fix [Jobs] Missing scroll bar in jobs UI logs (#1965)

mavdryk committed:
Fix [Batch run] Mandatory parameters should be selected v2 (#1968)

illia-prokopchuk committed:
Fix [Feature Sets Panel] Checking "Passthrough" unchecks "Online" target (#1970)

illia-prokopchuk committed:
Fix [Batch Run] Number input for Max iterations, Max errors and Parallel runs is malfunctioning (#1971)

ilan7empest committed:
Bump DRC version to `1.4.2` (#1973)

illia-prokopchuk committed:
Fix [Search] The search item list closing on scroll (#1960)

EZheln committed:
QA Sprint 147 done (#1975)

* Updated tests related to 1.5

1. added tests to check Version tag is filled when editing
- artifacts/dataset/model details view MLD009, MLA009, MLM020
- Jira: ML-4004

2. Check Minimum CPU value on Batch Run wizard
- Batch Run wizard - Resources
- Jira: ML-4308

3. Check tip and warning messages in Volumes section on Batch Run wizard
- Batch Run wizard - Resources
- Jira: ML-4310

* Added test to ML Function feature
- Check requirements field in Code Accordion on Create New Function page
- Jira: ML-4009

* Added test to Jobs And Workflows feature, update mock API with hub
functions
1. Check redirect to project`s Function Infopane from Job Overview
- current project
- Jira: ML-4332

2. Check mandatory of Container and Resource Path fields for V3IO volume
- Resources section in Batch Run
- Jira: ML-4355

3. Check retrieve the hub items catalog, function yaml
- Jira: ML-4450, ML-4500

* Added test to Datasets, Artifacts, Models feature, update mock API with tag requests
1. Check that version tag dropdown shows all tags on filters (Datasets, Artifacts, Models pages)
- Jira: ML-4385
2. Added update and delete possibility for tags on mock

* Added test to Batch Run Data input section on Jobs And Workflows feature
1. Check autocomplete MLRun Store path for datasets, artifacts, models in Batch Run wizard on Data input
- Jira: ML-4288, ML-4520

* Added tests to Projects and Function feature
1. Check Search projects is case insensitive
- Jira: ML-4378
2. Check visibility of Expand button
- Jira: ML-4121
3. Check function name validation
- should not contain uppercase letters, underline
- Jira: ML-4321, ML-4533
4. Check "Export YAML" action
- Jira: ML-4433

* Added test to Jobs And Workflows feature
1. Check setting schedule for a job
- Jira: ML-4293
2. Check changing the type of Environment variables in Batch Run
- Jira: ML-4228

---------

Co-authored-by: Olena Zhelnytska <olena.zhelnytska@medtronic.com>

ilan7empest committed:
Bump DRC version to `1.4.3` (#1981)

illia-prokopchuk committed:
Fix [Jobs] Monitor job pods view use too broad api call (#1976)

mariana-furyk committed:
Fix [Batch run] Labels issues on Bach run wizard (#1982)

mariana-furyk committed:
Fix [Projects] MLRun fails to delete project on the backend, but shows success in UI (#1836)

illia-prokopchuk committed:
Fix [Consumer groups] Table width (#1986)

illia-prokopchuk committed:
Fix [Jobs] print 400 error message when running local job (#1989)

mariana-furyk committed:
Fix [Batch run] Selected function from the project list hint the same function from the hub (#1992)

illia-prokopchuk committed:
Fix [Feature Sets] Passthrough popup appears (#1991)

mariana-furyk committed:
Fix [Batch Inference] The wrong iMessage for dataset input path (#1997)

mariana-furyk committed:
Fix [Jobs] Incorrect runs list is loaded after trying to load 'Heavy' runs (#1999)

ilan7empest committed:
Fix [Jobs] missing error reason when job abort fails (#1995)

ilan7empest committed:
Impl [Notifications] Add closing/removing manually (#1996)

ilan7empest committed:
Remove constants from jobs.util.js

mavdryk committed:
Fix [Batch run] Show function's inputs on re-run and mark mandatory items (#2003)

illia-prokopchuk committed:
Fix [Batch run] sort function names a-z (#2006)

illia-prokopchuk committed:
Fix [Jobs] Go back to list redirects to Overview page (#2008)

mariana-furyk committed:
Fix [Filter] Add default "Apply" button (#2009)

ilan7empest committed:
Fix [ML functions] Function without a status is in a 'Creating' status (#1994)

mariana-furyk committed:
Fix [Batch inference] The model_path input only supports store paths (#2012)

illia-prokopchuk committed:
Fix [Batch Run] App crashes on an attempt to delete Data input (#2014)

ilan7empest committed:
Bump DRC version to 1.5.0 (#2019)

mavdryk committed:
Fix [Batch run] Require mandatory data inputs and parameters before running the job (#2016)

illia-prokopchuk committed:
Fix [Functions panel] The "Save" and "Deploy" buttons shouldn't be enabled (#2021)

ilan7empest committed:
Impl [Navigation] Change project monitoring to be the default project view (#1984)

mariana-furyk committed:
Impl [Quick Links] Revamp tables quick links and action icons (#2022)

ilan7empest committed:
Fix [Project Monitor] mlrun long run - GET to mlrun/api/v1/runs?.. fails with 504 (#2023)

ilan7empest committed:
Bump DRC version to `1.5.1` (#2030)

mariana-furyk committed:
Impl [Batch Inference] Add Tech preview text (#2025)

mariana-furyk committed:
Fix [Navigation] Adjust Route to new path (#2028)

mariana-furyk committed:
Impl [Monitor Jobs] Add option to delete jobs in the UI (#2027)

mavdryk committed:
Fix [Batch run] "Next" button is disabled when hyperparams selected from file (#2032)

ilan7empest committed:
Fix [Project monitor] change quotes to parenthesis (#2033)

mariana-furyk committed:
Fix [Table] Bug with scroll (#2035)

illia-prokopchuk committed:
Fix [UI] Project monitoring - wrong filter for running jobs (#2036)

mariana-furyk committed:
Fix [Models/Datasets/Artifacts] Errors occur on opening actions menu (#2038)

EZheln committed:
QA Sprint 148 done (#2037)

* Added test to Jobs And Workflows feature
- Check click on default interval on simple scheduler in Advanced section
- Jira: ML-4426

* Updated tests on Batch Run feature
- Check next to the side step should be enabled in Batch Run
- Jira: ML-4633

* Updated test for Jobs And Workflows feature
1. Check redirection to function list from job Overview after deleting ML function
- Jira:ML-4335, 4501
2. Check Batch-Run running after edit GPU limit in Resources section
- Jira: ML-4429
3. Check "Max Iterations", "Max errors" inputs field availability according to the strategy type in Hyperparameter strategy
- Jira: ML-4452

* Updated test for Batch Run wizard
1. Check "Custom" strategy is removed as a Hyperparameter strategy in Batch Run
- Jira: ML-4507
2. Check back navigation from Job overview to Jobs Monitor tab and forward to Job overview
- Jira: ML-4416
3. Check components in Parameters section on Batch Run wizard with checked Hyper
- Jira: ML-4508
4. Check the "handler"/"method" option is not missing in Batch Run
- Jira: ML-4446
5. Updated autocomplete without tags MLRun Store path for feature vectors - Batch Run - Data input
- Jira: ML-4288, ML-4520
6. Check that version tag has "Click to add" status when it's empty after edited
- Jira: ML-4159
7. Check labels validations upon Batch Re-Run
- Jira: ML-4251, 4644

* - Updated function delete tags on mock
- Added test for changing "Method" after "Hyperparameter" check in Run Details (Jira: ML-4456)

* - Updated function delete tags on mock
- Added test for changing "Image name" field in Run Details section of Batch Run (Jira: ML-4511, ML-4606)

---------

Co-authored-by: Olena Zhelnytska <olena.zhelnytska@medtronic.com>

mariana-furyk committed:
Fix [ScheduleJobs/Model Endpoints] UI craches with error (#2042)

ilan7empest committed:
Fix [Batch Run] UI should not try to fetch hub functions without navigating to "Hub" tab (#2040)

ilan7empest committed:
Bump DRC to v`1.6.0` (#2045)

ilan7empest committed:
Fix [UI] add data-testid for buttons in models/artifacts/datasets screens (#2043)

illia-prokopchuk committed:
Impl [Batch-run] improved handling of parameters (#2044)

ilan7empest committed:
Fix [Project monitoring, Quick actions] Change instance name capitalization (#2046)

ilan7empest committed:
Fix [Models] Actions aren't aligned (#2047)

ilan7empest committed:
Bump DRC version to `1.7.0` (#2054)

mavdryk committed:
Impl [Wizards] should work as a single logical screen (#2050)

mariana-furyk committed:
Impl [Artifacts] Move Overview "Sources" according to Figma (#2048)

mariana-furyk committed:
Fix [Action menu] App crash when trying to 'edit' an ML function (#2053)

mariana-furyk committed:
Impl [Download]: Revamp Download by Figma (#2049)

mariana-furyk committed:
Fix [General] Issue with UI build job (#2055)

illia-prokopchuk committed:
Fix [Batch Run] Show function's inputs on re-run (#2057)

mariana-furyk committed:
Revamp [Models/Datasets/Artifacts] change UI according to Figma new design (#2058)

EZheln committed:
QA Sprint 149 done (#2056)

* Updated test for navigation bar
- project monitoring the main screen
- quick-actions the second screen
- Jira: ML-4667

* Created tests for Datasets, Artifacts, Models
- Check filter by "All" tag is performed when version tag was edited
- Jira: ML-4158

* Updated/created tests for Batch Inference
- update mock with batch inference_v2
- create test for check components (batch inference_v2, preview text, model path type store)
- Jira: ML-4485, ML-4781, ML-4697

---------

Co-authored-by: Olena Zhelnytska <olena.zhelnytska@medtronic.com>

EZheln committed:
QA Sprint 150 done (#2074)

* Update ts for HUB functions in Batch Run
- Categories mapping in "Filter by category" of HUB functions in Batch Run
- Jira: ML-4677
- "Batch-inference" is not shown among Hub functions in Function Selection section of Batch Run
- Jira: ML-4580
- "Apply" button in "Filter by category" of HUB functions in Batch Run
- Jira: ML-4294ests for HUB functions in Batch Run
- Categories mapping in "Filter by category" of HUB functions in Batch Run
- Jira: ML-4677
- "Batch-inference" is not shown among Hub functions in Function Selection section of Batch Run
- Jira: ML-4580
- "Apply" button in "Filter by category" of HUB functions in Batch Run
- Jira: ML-4294

* Create test for Feature Store
- Check type Redis in online types of Target Store section of Create Set
- Jira: ML-4066, ML-4277

* Updated tests for Datasets
- UI changes  according to Figma new design (register button, download icon)
- Jira: ML-3984, ML-4215

---------

Co-authored-by: Olena Zhelnytska <olena.zhelnytska@medtronic.com>

ilan7empest committed:
merge branch from upstream tests

illia-prokopchuk committed:
Fix [Project settings] Fix members tab display logic (#2073)

mavdryk committed:
Fix [Project] Validation on project's description in UI (#2076)

ilan7empest committed:
Fix [Quick actions] Add breadcrumbs to Quick actions page (#2077)

mavdryk committed:
Fix [New function] No default for handler (#2078)

ilan7empest committed:
Fix [UI] feature set usage example - missing import command & project value (#2079)

mavdryk committed:
Fix [Functions] Default handler is not displayed for functions (#2080)

ilan7empest committed:
Fix [Feature Store] Capitalize the first letter of the sentence in the Schema description. (#2081)

illia-prokopchuk committed:
Fix [Functions panel] Show "Volumes" dropdown on function edit (#2083)

illia-prokopchuk committed:
Fix [Project settings] does not present Default image of a project (#2085)

mavdryk committed:
Fix [Batch run] Disable version selection on function run (#2086)

illia-prokopchuk committed:
Fix [UI] presentation long-waiting UI operations (#2084)

mavdryk committed:
Fix [Batch run] rename method to handler (#2082)

ilan7empest committed:
Fix [Breadcrumbs] page crash when project list is not loaded (#2089)

ilan7empest committed:
Fix [UI] create feature set - schema entities tool tip is not red (#2088)

mariana-furyk committed:
Impl [Artifact/Models/Datasets] Add option to delete artifacts , models, dataset in the UI (#2039)

mariana-furyk committed:
Impl [Models] Wrong preview presentation (#2090)

mavdryk committed:
Fix [Batch run] UI crash on attempt to schedule job (#2091)

mariana-furyk committed:
Fix [Projects] an attempt to delete / archive project redirects to project screen (#2092)

mariana-furyk committed:
Fix [Functions Panel] clicking on deploy should not try to build a new image (#2094)

mariana-furyk committed:
Fix [Monitor jobs] UI fails on show details (#2098)

ilan7empest committed:
Fix [Batch run] remove the 'No data to show' (#2096)

ilan7empest committed:
Fix [Monitor workflows] Add a subtitle to monitoring workflow to indicate the list includes only run workflow (#2097)

mariana-furyk committed:
Fix [Functions Panel] After adding a function setting GPU, the function sometimes displays wrong GPU value (#2099)

ilan7empest committed:
Bump DRC version `1.8.0` (#2107)

ilan7empest committed:
Fix [UI] add the 'data-testid' for 'Add volume' and 'Add node selector' buttons (#2087)

mavdryk committed:
Fix [General] Increase cancel timeout, fix error notifications (#2093)

mariana-furyk committed:
Fix [Models] Model deploy in the UI does not impact (#2095)

mariana-furyk committed:
Fix [Functions Panel] function create - deploy button (#2100)

mariana-furyk committed:
Fix [Jobs] UI crash on 'pathname' (#2104)

ilan7empest committed:
Fix [UI] add data-testid for elements in batch run wizard (#2105)

ilan7empest committed:
Impl [Functions] Move create/edit 'function panel' to UI Demo mode (#2106)

mariana-furyk committed:
Fix [Delete job] issues with job delete (#2102)

mariana-furyk committed:
Fix [Project Settings] UI does not display pull_at_runtime project setting (#2103)

EZheln committed:
QA Sprint 151 done (#2101)

* Updated tests for Feature Store
- updated tests according to Revamp UI changes
- changed "Apply_Changes_Button" and "Cancel_Button" implementation on Feature Store Info Pane
- Jira: ML-4215

* Updated tests for Dataset
- changed Register button locator
- changed input hint for validation rules
- updated func deleteTags, putTags on mock
- Jira: ML-4835

* Updated tests for Feature Store page

---------

Co-authored-by: Olena Zhelnytska <olena.zhelnytska@medtronic.com>

mariana-furyk committed:
Fix [Models] Wrong preview presentation Part 1 - fixed bugs with click on name (#2109)

mavdryk committed:
Fix [Batch Run] Does not adjust based on function version (#2110)

illia-prokopchuk committed:
Impl [UI] Use the new artifacts v2 endpoints (#1882)

illia-prokopchuk committed:
Fix [UI] "Members" tab is not present for user with "project security admin" policy (#2112)

mavdryk committed:
Fix [Batch Run] missing parameters and data inputs for Hub functions (#2113)

mariana-furyk committed:
Fix [Feature Store] UI craches after checking the Partition checkbox (#2115)

mariana-furyk committed:
Fix [Feature Store] "Something went wrong." screen when add Feature Vector (#2116)

illia-prokopchuk committed:
Fix [Train model] auto select "auto-trainer" function (#2117)

illia-prokopchuk committed:
Revert [UI] Use the new artifacts v2 endpoints (#2119)

mavdryk committed:
Fix [Batch run] parameters values are deleted after updating type (#2118)

ilan7empest committed:
Bump DRC version to `1.8.1` (#2126)

ilan7empest committed:
Fix [Artifacts] Table header is hidden during table scroll (#2123)

mavdryk committed:
Fix [Batch run] unable to rerun the job (#2121)

mavdryk committed:
Fix [Feature Store] Features entity creation (#2122)

mavdryk committed:
Fix [Batch run] Incorrect completion message (#2124)

EZheln committed:
QA Sprint 152 done (#2114)

* Updated tests according to Revamp Datasets
- Quick links action menu list
- Download notification pop-up
- Jira: ML-3983

* Updated tests according to Revamp Artifacts, Datasets
- Quick links action menu list
- Preview in Quick links
- Download notification pop-up
- updated getArtifacts func on mock
- Preview modal
- Jira: ML-3983

* Updated tests according to new requirements
- Datasets, Artifacts, Models - delete options, Register, Train model
- Feature store - change capitalization name in breadcrumbs6
- Feature store - update tests after project monitoring become the default project view
- Jira: ML-4506, ML-4990, ML-4667, ML-3984

---------

Co-authored-by: Olena Zhelnytska <olena.zhelnytska@medtronic.com>

mariana-furyk committed:
Fix [Monitor Workflows] UI crash on attempt to browse "Inputs" tab (#2128)

illia-prokopchuk committed:
Impl [Batch Run] Add "Current project" to MLRun Store Path Scheme autocoplete (#2129)

illia-prokopchuk committed:
Fix [Model Train] display dataset URL conditionally (#2130)

mavdryk committed:
Fix [Build logs] Logs refresh is blinking too fast (#2131)

mavdryk committed:
Fix [General] Increase cancel request timeout from 60sec to 2min (#2132)

illia-prokopchuk committed:
Fix [Datasets][Preview] data displaying issue (#2133)

illia-prokopchuk committed:
Impl [UI] Use the new artifacts v2 endpoints (#2120)

mavdryk committed:
Impl [Register artifact] does not allow to provide tag (#2111)

mavdryk committed:
Fix [Artifact preview] Artifacts with "sql" extension are not previewed (#2135)

EZheln committed:
QA Sprint 153 done (#2139)

* Updated tests according to requirements
- Models, Artifacts, Datasets - added check Preview, Deploy button
- Models, Artifacts, Datasets - added check Pop_Out_Button on Preview tab
- Models, Artifacts, Datasets - added check delete options state
- Models - added overview Producer and Sources Headers
- Register Models - added new puth input implementation
- mock - updated postArtifact func with model_file param
- Jira: ML-4506

* Updated tests according to requirements
- redirection to Project monitoring page
- redirection to Quick actions page
- mock - created deleteRun func according to new delete jobs API
- mock - created deleteArtifact func according to new delete artifacts API
- mock - created getIguazioSelf func
- Jira: ML-5161, ML-4667

---------

Co-authored-by: Olena Zhelnytska <olena.zhelnytska@medtronic.com>

illia-prokopchuk committed:
Fix [Train model] auto select "auto-trainer" function with "train" handler (#2136)

mavdryk committed:
Fix [Breadcrumbs] The projects pages dropdown doesn't open up after model/dataset tag editing (#2137)

illia-prokopchuk committed:
Fix [Train model] validation fails for auto populated dataset (#2138)

ilan7empest committed:
Fix [Project monitoring] UI crash upon click on "Register artifact" via "Create new" dropdown (#2141)

illia-prokopchuk committed:
Fix [Batch Re-run] data inputs should be checked if they were used in running the job (#2142)

illia-prokopchuk committed:
Fix [Artifacts Preview] Wrong table size (#2143)

illia-prokopchuk committed:
Fix [Monitor Jobs] App crashes on  Preview icon click (#2144)

mavdryk committed:
Fix [Batch Run] change map to dict in parameters types (#2145)

mavdryk committed:
Fix [Batch run] Align "function templates" with the "functions hub" (#2146)

mavdryk committed:
Fix [Artifacts] Change text color of disabled Delete button to gray (#2148)

mavdryk committed:
Fix [Batch run] UI crash with "getBoundingClientRect" (#2149)

mariana-furyk committed:
Fix [Models] data displaying issue (#2150)

mariana-furyk committed:
Fix [Jobs] Delete job from the list view deletes single execution (#2151)

mavdryk committed:
Impl [Datasets] label_column should be added to Dataset details panel (#2155)

mavdryk committed:
Fix [Quick actions] crash upon click on Register artifact/dataset (#2156)

mavdryk committed:
Fix [ML functions] Function selection issue with same hash (#2157)

mariana-furyk committed:
Fix [Feature Vectors] Error after expanding Feature Vectors item (#2154)

mariana-furyk committed:
Fix [Jobs] "Delete" option should be hidden in the Action menu , when the job has status: Running/Pending (#2153)

mariana-furyk committed:
Fix [Feature sets] UI crash on attempt to preview feature set (#2152)

ilan7empest committed:
Fix [Workflows] Cannot read properties of null (reading toLowerCase) on attempt to filter workflows by status "All" (#2158)

ilan7empest committed:
Impl [Model endpoint] there should be a link to open Grafana and view full details (#2159)

ilan7empest committed:
Fix [Artifacts] Show actions when table as horizontal scroll (#2160)

ilan7empest committed:
Fix [Feature Vectors] UI crash upon click on Delete feature vector (#2162)

ilan7empest committed:
Fix [Feature Vectors] UI crash upon searching by entity during feature vector creation (#2163)

ilan7empest committed:
Fix [Feature Vectors] UI crash upon expanding feature during feature vector creation (#2164)

illia-prokopchuk committed:
Impl [Artifacts] New artifact v2 api for artifact delete (#2165)

EZheln committed:
QA Sprint 154 done (#2166)

* Updated tests according to requirement Batch Run as a single screen, delete jobs option
- updated tests - changed action menu options for Ready, Error, Running, Pending job states
- updated tests - changed Overview headers on Jobs monitor info pane
- added tests for Delete confirmation pop-up
- updated tests - check Batch Run steps visible/enable
- Jira: ML-4664

* Updated tests according to requirement
- updated tests - added check the notification pop-up on Feature vector creation
- updated tests on Feature Store page
- updated tests on Jobs and Workflows page
- mock - updateed new artifacts v2 endpoints
- mock - updated register artifact postArtifact func
- mock - updated deleteArtifact func
- Jira: ML-4196, ML-4916

---------

Co-authored-by: Olena Zhelnytska <olena.zhelnytska@medtronic.com>

ilan7empest committed:
Fix [Artifacts, Datasets, Models] successful deletion notification message (#2169)

Taras-Hlukhovetskyi committed:
Fix [Filter] Tag list doesnt scroll in Filter by dialog (#2171)

ilan7empest committed:
Bump DRC version `1.8.2` (#2179)

mavdryk committed:
Fix [Artifacts] missing delete confirmation message (#2147)

ilan7empest committed:
Fix [Monitor jobs] Unnecessary context: parameter presents in UI (#2161)

ilan7empest committed:
Fix [Batch run] fix labels validation (#2168)

mavdryk committed:
Fix [Datasets] Change notification message on removing the tag (#2174)

Taras-Hlukhovetskyi committed:
Fix [Quick actions] UI crash upon click on Register model, Define alerts, Run workflow or Create workflow (#2175)

Taras-Hlukhovetskyi committed:
Fix [Input] placeholder is not aligned to center vertically (#2176)

mavdryk committed:
Fix [Jobs] log is being duplicated upon refresh (#2177)

Taras-Hlukhovetskyi committed:
[Fix [Models] Page stuck in a loading state (#2178)

mariana-furyk committed:
Fix [Consumer groups] Table is empty although BE response is not empty (#2172)

Taras-Hlukhovetskyi committed:
Fix [Artifacts] Page stuck in a loading state (#2181)

pinis-gini-apps committed:
Fix [Table] Remove unnecessary class from table header (#2180)

illia-prokopchuk committed:
Fix [Dataset] Add indication to dataset preview (#2182)

illia-prokopchuk committed:
Fix [Project settings] Members screen - does not take into account the owner (#2183)

Taras-Hlukhovetskyi committed:
Fix [Feature store] uniformity of tag filter labels (#2184)

Taras-Hlukhovetskyi committed:
Fix [Model endpoints] Features analysis data missing (#2185)

mavdryk committed:
Fix [Artifacts] Change uniqueness validation and update messages (#2186)

EZheln committed:
QA Sprint 155 done (#2187)

- updated Dataset details panel with label_column
- added tests for Train model wizard on Datasets
- updated some tests Batch Run as a single logical screen
- mock - updated deleteArtifact func
- update func on mock for Train model auto-trainer function
-Jira: ML-5100, ML-1855, ML-4664

Co-authored-by: Olena Zhelnytska <olena.zhelnytska@medtronic.com>

Commit:
Updated tests for Train model wizard
- Jira: ML-5328, ML-5234, ML-5333

mariana-furyk commented on tests/mockServer/mock.js:
```diff
@@ -1165,21 +1165,18 @@ function putTags(req, res){
-      artifactForUpdate.tag = req.params.tag
-    }
+
+  if (artifactList){
```
what if artifactsList length = 0?

mariana-furyk commented on tests/mockServer/mock.js:
```diff
@@ -1190,20 +1187,19 @@ function putTags(req, res){
 
 function deleteTags(req, res){
   const collectedArtifact = artifacts.artifacts
```
```suggestion
  const collectedArtifacts = artifacts.artifacts
```

mavdryk commented on tests/mockServer/mock.js:
```diff
@@ -1190,20 +1187,19 @@ function putTags(req, res){
+    const artifactByTag = collectedArtifact.find(artifact => (artifact.metadata?.tag === req.params.tag || artifact.tag === req.params.tag))
+    remove(artifacts.artifacts, artifactByTag)
+  }
+  else if (collectedArtifact?.length === 1){
```
else if should be on the previous line

mavdryk replied:
there should a space before opening bracket `{`. Check other places.
Better to enable prettier and eslint

mariana-furyk commented on tests/mockServer/mock.js:
```diff
@@ -1190,20 +1187,19 @@ function putTags(req, res){
-    else if (collectedArtifact.tag === req.params.tag){
-      collectedArtifact.tag = ''
-    }
+    .filter(artifact => ((artifact.metadata?.project === req.params.project || artifact.project === req.params.project) 
```
define a const, like artiact = artifact.metadata ?? artifact adn use it where you need

mariana-furyk commented on tests/mockServer/mock.js:
```diff
@@ -1165,21 +1165,18 @@ function putTags(req, res){
   const projectName = req.params.project
 
-  const artifactForUpdate = artifacts.artifacts
+  const artifactList = artifacts.artifacts
```
use the same name for const ( artifactsList or collectedArtifacts) for the same data

Commit:
Updated tests according to requirements
- changed uniqueness validation and for Register artifact, Register dataset
- update messages for Register artifact, Register dataset
- updated mock server functions related to edit and delete tags acording to new implementation
- removed the label value validation rules Batch Run wizard
- changed message Tag to Version Tag feature store, feature vector
- added check label validation rules on Create new project pop-up
- Jira: ML-4927, ML-4916, ML-5440, ML-4835
